mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarouselV2.tsx

```diff
@@ -0,0 +1,110 @@
+import { Dimensions, Image, ImageSourcePropType, Platform } from 'react-native'
+import SwiperFlatList from 'react-native-swiper-flatlist'
+import ImageC from '@assets/images/onboarding/welcome-screen-a.png'
```

Comment:
let be consistent on the file and variable naming, e.g.:
```suggestion
import ImageA from '@assets/images/onboarding/welcome-screen-a-dark.png'
```